package/Makefile

```diff
@@ -15,7 +15,6 @@ ROOT_DIR := $(realpath $(WHEREAMI)/../)
 include $(ROOT_DIR)/make/system-id.mk
 
 TOOLS_DIR := $(ROOT_DIR)/tools
-BUILD_DIR := $(ROOT_DIR)/build
```

Comment:
BUILD_DIR was double defined before? Where is it coming from since it is used for PACKAGE_DIR below?